packages/mattermost-redux/src/types/posts.ts

```diff
@@ -146,3 +146,10 @@ export declare type OpenGraphMetadata = {
     url?: string;
     images: OpenGraphMetadataImage[];
 };
+
+export declare type PostPreviewMetadata = {
+    post_id: string;
+    post?: Post;
```

Comment:
Remind me, what's the reason for `post` being optional?

Reply:
I actually remove post from this object when storing the metadata in state because the post will already exist somewhere else in state. So that's why I marked it as optional